testpackage/tests/Flowthrough_damr/Flowthrough_damr.cfg

```diff
@@ -0,0 +1,113 @@
+ParticlePopulations = proton
+
+project = Flowthrough
+propagate_field = 1
+propagate_vlasov_acceleration = 1
```

Comment:
I suggest turning acceleration off - at least trialling if it still works?

Reply:
So that doesn't work quite right; instead of a L2 refinement following the pulse we get a L1 region at 100s and no refinement after, so that loses out on testing induced refinement at least.